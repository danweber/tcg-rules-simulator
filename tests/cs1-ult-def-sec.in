VERSION:2
# [Main] Until the end of your opponent's turn, all of your Monsters cannot be deleted in battle.
SECURITY1:CS1-UltimateDefense
FIELD1:Tiny1,CS1-02,REST Tiny2,CS1-02,REST
FIELD2:Attack1,DW1-Van Attack2,DW1-Van Attack3,DW1-Van Attack4,DW1-Van
HAND2:DW1-09
SECURITY1:CS1-04 CS1-UltimateDefense
GAME:P2 T3 MAIN 8
DUMP:SECURITY
ECHO: -- P2 attacks, reveals security a thing
CMD:ATTACK Attack1 Player
STEP:AUTOANSWER
DUMP:SECURITY,DP

#EXIT

ECHO: -- P2 attacks, can't kill a thing
CMD:ATTACK Attack2 Tiny1
DUMP:SECURITY,DP

#EXIT

ECHO: -- P2 attacks, reveals CS1-04
CMD:ATTACK Attack3 Player
DUMP:SECURITY,DP

ECHO: -- sit it down...
CMD:PLAY DW1-Sitdown
STEP:AUTOANSWER:-1:CAPTURE
DUMP:SECURITY,DP

ECHO: -- and attack into it, with nothing happening
CMD:ATTACK Attack4 I7
DUMP:SECURITY,DP





