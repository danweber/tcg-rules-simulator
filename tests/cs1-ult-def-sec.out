FIELD1:Tiny1,CS1-Papershield,REST Tiny2,CS1-Papershield,REST
FIELD2:Attack1,DW1-Vanillamon Attack2,DW1-Vanillamon Attack3,DW1-Vanillamon Attack4,DW1-Vanillamon
SECURITY1:CS1-Stoneshield CS1-Ultimatedefense
SECURITY2:
GAME:P2 T3 MAIN 8
ECHO: -- P2 attacks, reveals security a thing
FIELD1:Tiny1,CS1-Papershield,REST,500DP Tiny2,CS1-Papershield,REST,500DP
FIELD2:Attack1,DW1-Vanillamon,REST,5000DP Attack2,DW1-Vanillamon,5000DP Attack3,DW1-Vanillamon,5000DP Attack4,DW1-Vanillamon,5000DP
SECURITY1:CS1-Stoneshield
SECURITY2:
GAME:P2 T3 MAIN 8
ECHO: -- P2 attacks, can't kill a thing
FIELD1:Tiny1,CS1-Papershield,REST,500DP Tiny2,CS1-Papershield,REST,500DP
FIELD2:Attack1,DW1-Vanillamon,REST,5000DP Attack2,DW1-Vanillamon,REST,5000DP Attack3,DW1-Vanillamon,5000DP Attack4,DW1-Vanillamon,5000DP
SECURITY1:CS1-Stoneshield
SECURITY2:
GAME:P2 T3 MAIN 8
ECHO: -- P2 attacks, reveals CS1-04
FIELD1:Tiny1,CS1-Papershield,REST,500DP Tiny2,CS1-Papershield,REST,500DP I7,CS1-Stoneshield,4500DP
FIELD2:Attack1,DW1-Vanillamon,REST,5000DP Attack2,DW1-Vanillamon,REST,5000DP Attack3,DW1-Vanillamon,REST,5000DP Attack4,DW1-Vanillamon,5000DP
SECURITY1:
SECURITY2:
GAME:P2 T3 MAIN 8
ECHO: -- sit it down...
FIELD1:Tiny1,CS1-Papershield,REST,500DP Tiny2,CS1-Papershield,REST,500DP I7,CS1-Stoneshield,REST,4500DP
FIELD2:Attack1,DW1-Vanillamon,REST,5000DP Attack2,DW1-Vanillamon,REST,5000DP Attack3,DW1-Vanillamon,REST,5000DP Attack4,DW1-Vanillamon,5000DP
SECURITY1:
SECURITY2:
GAME:P2 T3 MAIN 4
ECHO: -- and attack into it, with nothing happening
FIELD1:Tiny1,CS1-Papershield,REST,500DP Tiny2,CS1-Papershield,REST,500DP I7,CS1-Stoneshield,REST,4500DP
FIELD2:Attack1,DW1-Vanillamon,REST,5000DP Attack2,DW1-Vanillamon,REST,5000DP Attack3,DW1-Vanillamon,REST,5000DP Attack4,DW1-Vanillamon,REST,5000DP
SECURITY1:
SECURITY2:
GAME:P2 T3 MAIN 4
